polaris/polaris/tests/processes/test_create_stellar_deposit.py

```diff
@@ -169,12 +172,59 @@ def test_deposit_stellar_no_trustline(acc1_usd_deposit_transaction_factory):
     `create_stellar_deposit` sets the transaction with the provided `transaction_id` to
     status `pending_trust` if the provided transaction's Stellar account has no trustline
     for its asset. (We assume the asset's issuer is the server Stellar account.)
+
+    This is the flow when a wallet/client submits a deposit without the sending
+    "claimable_balance_supported=True" in their POST request body.
+    or sending "claimable_balance_supported=False" in their POST request body.
+    Such that the deposit transaction is blocked
+    until the Wallet user creates a trustline to that asset
     """
     deposit = acc1_usd_deposit_transaction_factory()
-    deposit.status = Transaction.STATUS.pending_anchor
+    deposit.status = Transaction.STATUS.pending_trust
```

Comment:
I actually wouldn't make the transaction's status `pending_trust` because that has historically meant the wallet needs to establish the trustline and the anchor is blocked. But `pending_anchor` indicates that the anchor is the one who has things to do and the wallet can sit tight.

Reply:
Ah ok I can revert this then.
I think the docstring threw me off a bit.